rpc_jobs/standard_job_premerge.yml

```diff
@@ -82,6 +83,24 @@
 
       String credentials = "{credentials}"
 
+      // We need to checkout the repo on the CIT Slave so that
+      // we can check whether the patch only includes changes
+      // in the skip list. If that is the case, we can skip
+      // the rest of the job execution and return success.
+      common.shared_slave() {{
+        common.withRequestedCredentials(credentials) {{
+          stage('Check PR against skip-list') {{
+            print("Triggered by PR: ${{env.ghprbPullLink}}")
+            common.clone_with_pr_refs(
+              "${{env.WORKSPACE}}/${{env.RE_JOB_REPO_NAME}}",
+            )
+            if (common.skip_list_test("{skip_list}")) {{
```

Comment:
common.skip_list_test expects a list of strings, but this looks like it will present the whole list as a single string?

Reply:
Is there a better way to make it pass the things correctly? Or perhaps to change the function to receive the things JJB is able to pass. I tried making JJB use an object here and it croaked.